src/utils/constants/links.ts

```diff
@@ -12,7 +16,7 @@ const PAGES = Object.freeze({
   DASHBOARD_ISSUE_REQUESTS: '/dashboard/issue-requests',
   DASHBOARD_REDEEM_REQUESTS: '/dashboard/redeem-requests',
   DASHBOARD_RELAY: '/dashboard/relay',
-  VAULT: '/vault',
+  VAULT: `/vault/:${URL_PARAMETERS.VAULT_ADDRESS}`,
```

Comment:
@tomjeatt 
I used dynamic routing but I did not want to hardcode the URL parameters. I tried and managed via a variable.
I used `history.push` rather than `Link` component in the table because using `Link` required style changes in the table. I think we can handle it via another ticket.

Reply:
Ok that sounds fine - you shouldn't need to hardcode the URL parameters but as you suggest this is a bigger issue and it sits outside the scope of this this ticket. I'll put a ticket in the backlog to look at dynamic routing across the application. The approach you've taken is fine for a single instance - it only really becomes an issue if we need to scale it up and since we don't have any other upcoming features which require dynamic routing we can kick this down the road for now.